models/org.go

```diff
@@ -224,23 +225,55 @@ func DeleteOrganization(org *User) (err error) {
 		return err
 	}
 
-	if err = deleteBeans(sess,
-		&Team{OrgID: org.ID},
-		&OrgUser{OrgID: org.ID},
-		&TeamUser{OrgID: org.ID},
+	if err = deleteOrg(sess, org); err != nil {
+		return err
+	}
+
+	if err = sess.Commit(); err != nil {
+		return err
+	}
+
+	return RewriteAllPublicKeys()
```

Comment:
@lunny [Why did you add it then](https://github.com/appleboy/gitea/commit/ba134bd27abe099efcd748773c410b3aa78521fc#diff-97681c8a2932ea3c23325d3518374d98R239)? :stuck_out_tongue_winking_eye: